apps/mull-ui/src/app/components/navigation/sub-nav-bar/chat-header.tsx

```diff
@@ -11,10 +9,7 @@ export interface ChatHeaderProps {
 
 export const ChatHeader = ({ style, className, eventTitle }: ChatHeaderProps) => {
   return (
-    <div className={`header ${className}`} style={style}>
-      <button className="hamburger-button">
-        {<FontAwesomeIcon icon={faBars} size="2x" color="grey" className="hamburger-menu" />}
-      </button>
+    <div className={`header ${className ? className : ''}`} style={style}>
```

Comment:
Styling needs to accommodate long event names
![image](https://user-images.githubusercontent.com/33991121/112904230-a0229280-90b6-11eb-938b-91ffa1f6ed1a.png)

Reply:
![image](https://user-images.githubusercontent.com/33991121/113462223-ded58700-93ed-11eb-8a2b-c839806c06aa.png)
Needs a bit more padding